packages/cli/package.json

```diff
@@ -126,7 +128,8 @@
     "prepublishOnly": "pnpm run build"
   },
   "dependencies": {
-    "@prisma/engines": "workspace:*"
+    "@prisma/engines": "workspace:*",
+    "jose": "^5.2.2"
```

Comment:
Is this in the correct place? All the other things seem to be in `devDep`.

Reply:
Its not that complex, let's inline it.

https://github.com/panva/jose/blob/05c4351be3a356da2a0e882fbbd8006f2725ec7b/src/util/decode_jwt.ts#L22

Reply:
If you move it to `devDepenbdencies` we will bundle it automatically. And if it has an ESM export then we would also tree-shake the unused imports. If it's small enough you don't need to worry about inlining.